src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderBooking.java

```diff
@@ -154,6 +160,20 @@ public void add(final OffenderCourtCase courtCase) {
         courtCase.setOffenderBooking(this);
     }
 
+    public void add(final ProfileType profileType, final ProfileCode code) {
```

Comment:
Does this have test coverage? I couldn't see any.

Reply:
yes, via the booking creation apis